Editor/Deployment/DeploymentSettings.cs

```diff
@@ -96,15 +106,15 @@ public int ScenarioIndex
             && IsBuildFilePathFilled;
 
         public bool IsBuildFilePathFilled => IsValidScenarioIndex
```

Comment:
Why does this Index still matter, now that we're using the enum? How are we loading the scenarios? I'm probably failing to understand something here.